cmd/cluster-kube-controller-manager-operator-tests-ext/main.go

```diff
@@ -76,6 +76,10 @@ func runGinkgoTest(ctx context.Context, testName string) *extensiontests.Extensi
 	suiteConfig, reporterConfig := ginkgo.GinkgoConfiguration()
 	suiteConfig.FocusStrings = []string{escapeRegexChars(testName)}
 
+	// Configure JUnit reporter for CI integration
+	reporterConfig.JUnitReport = "junit.xml"
+	reporterConfig.JSONReport = "report.json"
```

Comment:
```suggestion
junitDir := flag.String("junit-dir", "", "Path to directory for junit.xml report")
flag.Parse()

if *junitDir != "" {
    junitPath := filepath.Join(*junitDir, "junit.xml")
    reporterConfig.JUnitReport = junitPath
    reporterConfig.JSONReport = filepath.Join(*junitDir, "report.json")
}

```
by default → no junit output.
When CI runs → it passes --junit-dir=${ARTIFACT_DIR}.

in your ci-operator step,
```
- as: tests-extension
  commands: |
    echo "Build binary cluster-kube-controller-manager-operator-tests-ext"
    make tests-ext-build

    echo "Running ./cluster-kube-controller-manager-operator-tests-ext with sanity test"
    mkdir -p ${ARTIFACT_DIR}
    ./cluster-kube-controller-manager-operator-tests-ext run-test \
      --junit-dir=${ARTIFACT_DIR} \
      -n "[Jira:kube-controller-manager][sig-api-machinery] sanity test should always pass [Suite:openshift/cluster-kube-controller-manager-operator/conformance/parallel]"
  from: src

```

Reply:
Prow gathers them automatically because $ARTIFACT_DIR is always collected.